contrib/packs/actions/pack_mgmt/download.py

```diff
@@ -120,47 +70,43 @@ def _clone_repo(repo_url, verifyssl=True, branch='master'):
         Repo.clone_from(repo_url, abs_local_path, branch=branch, depth=1)
         return abs_local_path
 
-    def _move_packs(self, abs_repo_base, packs, abs_local_path, subtree):
+    def _move_pack(self, abs_repo_base, pack_name, abs_local_path):
```

Comment:
We move `.git`, so you'll be able to—I double-checked it. :)

One simple reason for cloning to temp first — if the pack is already there, we don't want to mess up the config (see L86 and below).